askama_parser/src/error.rs

```diff
@@ -1,6 +1,9 @@
+//! Parser error handling.
```

Comment:
Common style is that the first line of a comment does not end with a period (it's usually sentence fragment, not a full sentence).

Reply:
Done.